Ninjathulhu/Ninjathulhu.Game/Component.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace rogue.src.sim
```

Comment:
Let's put these in the `Ninjathulhu.Game` namespace, maybe `Ninjathulhu.Game.Components`